src/ontology/cl-edit.owl

```diff
@@ -29166,7 +29166,8 @@ AnnotationAssertion(terms:contributor obo:CL_4023011 <https://orcid.org/0000-000
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023011 "ILX:0770149")
 AnnotationAssertion(oboInOwl:inSubset obo:CL_4023011 cl:BDS_subset)
 AnnotationAssertion(rdfs:label obo:CL_4023011 "lamp5 GABAergic cortical interneuron")
-EquivalentClasses(obo:CL_4023011 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000032148)))
+EquivalentClasses(obo:CL_4023011 ObjectIntersectionOf(obo:CL_4023064 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000032148)))
```

Comment:
Changing the parental terms produces that many cells cannot be classified as this cell type anymore.